### DISCUSSION TITLE
Zooming in the numeric list

### DISCUSSION THREAD

**Discussion by Gaaruda** [vslinko/obsidian-zoom#12] [Archive]
An option to zoom inside the numeric list such as 
1. hello
       1. what
2. are you doing

**Comment by vslinko** [comment]
@KALKI-the-avataar Thank you for your contribution! Added you idea to the backlog https://github.com/users/vslinko/projects/2#card-62690540

**Reply by vslinko** [reply]
Released in https://github.com/vslinko/obsidian-zoom/releases/tag/0.1.2
